grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandler.java

```diff
@@ -31,6 +31,14 @@
 @UnstableApi
 public interface UnframedGrpcErrorHandler {
 
+    static UnframedGrpcErrorHandlerBuilder builder() {
+        return new UnframedGrpcErrorHandlerBuilder(UnframedGrpcStatusMappingFunction.of());
+    }
+
+    static UnframedGrpcErrorHandlerBuilder builder(UnframedGrpcStatusMappingFunction statusMappingFunction) {
+        return new UnframedGrpcErrorHandlerBuilder(statusMappingFunction);
```

Comment:
Sorry, but I've just realized that `UnframedGrpcStatusMappingFunction` is an optional property with default value (= `UnframedGrpcStatusMappingFunction.of()`). Could you remove this method and add a setter method to the builder instead?